index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+    <h1>Hello World, we are software developers!</h1>
+</body>
```

Comment:
- I think it would be better to enclose your ```<h1>``` tags inside a ```<header>``` tag for readability purposes and for better SEO(Search Engine Optimization). For example:
```
<body>
  <header>
    <h1>Hello World, we are software developers!</h1>
  </header>
</body>
```